src/main/java/book/store/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -41,24 +41,26 @@ public ShoppingCartDto addBookInShoppingCart(Long userId, CartItemRequestDto req
             return shoppingCartMapper.toDto(userShoppingCart);
         }
 
-        CartItem cartItem1 = userShoppingCart.getCartItems().stream()
+        CartItem cartItemWithoutNewQuantity = userShoppingCart.getCartItems().stream()
                 .filter(cartItemBook -> cartItemBook.getBook().getId()
                 .equals(requestDto.getBookId()))
                 .findFirst()
```

Comment:
these lines of code are duplicated. Do you need to get this cart item again here?